src/jit/codegenlinear.h

```diff
@@ -183,15 +183,18 @@
     void                genJmpMethod(GenTreePtr jmp);
 
 #if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-    void genGetStructTypeSizeOffset(const SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR& structDesc,
-                                    var_types* type0,
-                                    var_types* type1,
-                                    emitAttr* size0,
-                                    emitAttr* size1,
-                                    unsigned __int8* offset0,
-                                    unsigned __int8* offset1);
+    void getGetStructTypeOffset(const SYSTEMV_AMD64_CORINFO_STRUCT_REG_PASSING_DESCRIPTOR& structDesc,
```

Comment:
@CarolEidt I got comment on previous PR by @sivarv that the CC for naming method is to use gen prefix only if the method generates code. If it is a helper method that does not generate code the name should not have gen prefix. 
@sivarv This method is a helper method that is only used in codegenxarch.cpp. Localizing it definition limited to where it is used made sense.